responders/src/API/EMBC.Responders.API/Controllers/RegistrationsController.cs

```diff
@@ -346,16 +350,27 @@ public class RegistrationsMapping : Profile
     {
         public RegistrationsMapping()
         {
+            CreateMap<VerifySecurityQuestionsResponse, ESS.Shared.Contracts.Submissions.VerifySecurityQuestionsResponse>()
+                .ReverseMap()
+                ;
+
             CreateMap<SecurityQuestion, ESS.Shared.Contracts.Submissions.SecurityQuestion>()
+                .ForMember(d => d.AnswerIsMasked, opts => opts.MapFrom(s => s.Answer.Contains("*")))
```

Comment:
This is a dangerous assumption - the incoming DTO should probably have a bool AnswerChanged that the frontend can control